From: Bruno Haible <bruno@clisp.org>
Subject: tests: Avoid an autopoint-3 failure caused by an Autoconf 2.71 bug.
Origin: upstream, https://git.savannah.gnu.org/gitweb/?p=gettext.git;a=commit;h=f23b7847990cb31b59729517314e5d0fb649f64e
Forwarded: not-needed

    * gettext-tools/tests/autopoint-3: Remove the autom4te cache before invoking automake.

--- a/gettext-tools/tests/autopoint-3
+++ b/gettext-tools/tests/autopoint-3
@@ -123,6 +123,20 @@
 # error "configure.ac:3: error: required file './ltmain.sh' not found".
 touch ltmain.sh
 
+# Avoid an Automake 1.16.5 failure, in the presence of Autoconf 2.71, on an ext4
+# file system:
+#   PREFIX/share/automake-1.16/am/depend2.am: error: am__fastdepCC does not appear in AM_CONDITIONAL
+#   PREFIX/share/automake-1.16/am/depend2.am:   The usual way to define 'am__fastdepCC' is to add 'AC_PROG_CC'
+#   PREFIX/share/automake-1.16/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
+#   PREFIX/share/automake-1.16/am/depend2.am: error: AMDEP does not appear in AM_CONDITIONAL
+#   PREFIX/share/automake-1.16/am/depend2.am:   The usual way to define 'AMDEP' is to add one of the compiler tests
+#   PREFIX/share/automake-1.16/am/depend2.am:     AC_PROG_CC, AC_PROG_CXX, AC_PROG_OBJC, AC_PROG_OBJCXX,
+#   PREFIX/share/automake-1.16/am/depend2.am:     AM_PROG_AS, AM_PROG_GCJ, AM_PROG_UPC
+#   PREFIX/share/automake-1.16/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
+# The cause is described in
+# <https://lists.gnu.org/archive/html/bug-automake/2023-07/msg00020.html>.
+rm -rf autom4te.cache
+
 ${AUTOMAKE} -a -c >/dev/null 2>autopoint.err
 test $? = 0 || { cat autopoint.err; Exit 1; }
 
